build_files_updated_unittest.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
I think something went wrong with the merge here because in #10243, this file was recently deleted and cmake/update_file_lists.sh was added.